examples/tutorials/README.md

```diff
@@ -27,28 +30,48 @@ practitioners. By enabling the growth of open source tools for drug discovery,
-* [Part 18: Using Reinforcement Learning to Play Pong](18_Using_Reinforcement_Learning_to_Play_Pong.ipynb)
-* [Part 19: Large Scale Chemical Screens](19_Large_Scale_Chemical_Screens.ipynb)
-* [Part 20: ConvertingDeepChem Models to TensorFlow Estimators](20_Converting_DeepChem_Models_to_TensorFlow_Estimators.ipynb)
-* [Part 21: Introduction to Bioinformatics](21_Introduction_to_Bioinformatics.ipynb)
-* [Part 22: Using HuggingFace + Transfer Learning for Toxicity Predictions](22_Transfer_Learning_With_HuggingFace_tox21.ipynb)
+The tutorial is organized as follows:
 
+### Introduction to DeepChem
+The following tutorials covers the core features of DeepChem. Covering these
+tutorials will help you in getting started with DeepChem for machine learning. The later
+tutorials discuss about using DeepChem for specific applications.
 
+* [1 The Basic Tools of the Deep Life Sciences](The_Basic_Tools_of_the_Deep_Life_Sciences.ipynb)
+* [2 Working With Datasets](Working_With_Datasets.ipynb)
+* [3 An Introduction to MoleculeNet](An_Introduction_To_MoleculeNet.ipynb)
+* [4 Creating Models with Tensorflow and Pytorch](Creating_Models_with_TensorFlow_and_PyTorch.ipynb)
+* [5 Introduction to Graph Convolutions](Introduction_to_Graph_Convolutions.ipynb)
+* [6 Working With Splitters](Working_With_Splitters.ipynb)
+* [7 Advanced Model Training](Advanced_Model_Training.ipynb)
+* [8 Putting Multitask Learning to Work](Putting_Multitask_Learning_to_Work.ipynb)
+* [9 Conditional Generative Adversarial Networks](Conditional_Generative_Adversarial_Networks.ipynb)
+* [10 Training a Generative Adversarial Network on MNIST](Training_a_Generative_Adversarial_Network_on_MNIST.ipynb)
+* [11 Introduction to Model Interpretability](Introduction_to_Model_Interpretability.ipynb)
+* [12 Uncertainty in Deep Learning](Uncertainty_In_Deep_Learning.ipynb)
+* [13 Creating a high fidelity model from experimental data](Creating_a_high_fidelity_model_from_experimental_data.ipynb)
+* [14 Large Scale Chemical Screens](Large_Scale_Chemical_Screens.ipynb)
+* [15 Using Reinforcement Learning to Play Pong](Using_Reinforcement_Learning_to_Play_Pong.ipynb)
+
+### Molecular Machine Learning
+* [1 Molecular Fingerprints](Molecular_Fingerprints.ipynb)
+* [2 Going Deeper on Molecular Featurizations](Going_Deeper_on_Molecular_Featurizations.ipynb)
+* [3 Learning Unsupervised Embeddings for Molecules](Learning_Unsupervised_Embeddings_for_Molecules.ipynb)
+* [4 Synthetic Feasibility Scoring](Synthetic_Feasibility_Scoring.ipynb)
+* [5 Atomic Contributions for Molecules](Atomic_Contributions_for_Molecules.ipynb)
+* [6 Transfer Learning with ChemBERTa Transformers](Transfer_Learning_With_ChemBERTa_Transformers.ipynb)
+* [7 Training a Normalizing Flow on QM9](Training_a_Normalizing_Flow_on_QM9.ipynb)
+
+### Modeling Proteins
+* [1 Modeling Protein Ligand Interactions](Modeling_Protein_Ligand_Interactions.ipynb)
```

Comment:
Protein ligand interactions should likely be broken into their own top level section
```
### Protein Ligand Modeling
```
The techniques here are pretty different from those for proteins alone :)